db/write_callback_test.cc

```diff
@@ -144,24 +147,37 @@ TEST_F(WriteCallbackTest, WriteWithCallbackTest) {
 
               db_impl = dynamic_cast<DBImpl*>(db);
               ASSERT_TRUE(db_impl);
-
+              
+              // Writers that have called JoinBatchGroup.
+              std::atomic<uint64_t> threads_joined(0);
```

Comment:
Perhaps threads_joining is a better name. "Joined" gives the impression that the joining process has finished already while we are at the start of the join method.